Scripts/Miscellaneous/Spotify_Playlist_Generator/spotify_playlist_generator.py

```diff
@@ -0,0 +1,82 @@
+import argparse
+import os
+import random
+
+import spotipy
+from spotipy.oauth2 import SpotifyOAuth
+
+scope = "playlist-modify-public"
+
+# Create an instance of spotify library
+spotipy_instance = spotipy.Spotify(auth_manager=SpotifyOAuth(scope=scope, cache_path=os.getcwd()))
+
+
+# Method to read the command line arguments and parse them
+def get_args():
+    parser = argparse.ArgumentParser(description='Split a playlist into multiple playlists')
+    # Required arguments for the program
+    parser.add_argument('-p', '--playlist_id', required=True,
+                        help='Playlist ID')
+    parser.add_argument('-l', '--limit', required=True, default=20,
+                        help='Size of each small playlist')
+    return parser.parse_args()
+
+
+# This method returns all the Spotify Song IDs from a given playlist
+def get_track_ids_for_playlist(playlist):
+    res = []
+    for song in playlist:
+        #Extract the ID of the track
```

Comment:
when you are writing  single line command use after you statement
```suggestion
         res.append(song['track']['id']) # Extract the ID of the track
```